src/qt/res/README.md

```diff
@@ -0,0 +1,46 @@
+
+#### SVG Source File
+SVGs are used as source files because they can scale while retaining image quality but are not used in production due to limited application support.
+If different-sized production (PNG) icons are required, they can be generated from the associated SVG source file in any vector-based tool.
+
+#### PNG Production File
+PNGs are used in production due to wide application support, transparency support, and better image quality compared to competing file types
+such as JPEG.
+
+### Icon Grid
+Bitcoin Core uses an 8-point grid system for its icons which utilizes size increments of 8px to visually orientate elements within a grid. The majority of screen sizes are divisible by 8, which makes fitting elements easier using an 8-point grid system. This system makes things more efficient for designers and aids in maintaining consistency across applications.
+
+### Icon Style
+Icon style should adhere to the following principles to maintain consistency and clarity:
+
+- To ensure icons look good at any scale, a minimalistic design should be used.  
+- Icon size should be 32x32px by default. A different sized icon may be necessary under special circumstances. If the 32x32px size is too small for the required icon placement, then the icon should be scaled up or down while adhering to the 8-point grid system.
+- The icon should avoid organic shapes or elements, these do not scale well.
+- Colors should be consistent with existing Bitcoin Core icons.
+- Icons should adhere to the 8-point grid system (see above).
+
+### Optimizing Production Files (PNG):
+Production (PNG) files must be processed by the [optimize-png.py](https://github.com/bitcoin-core/bitcoin-maintainer-tools/blob/master/optimise-pngs.py) script before their inclusion in Bitcoin Core. PNG optimization removes various unnecessary color profiles, ancillary (alla), and text chunks, resulting in a lossless reduction of the file's size.
+
+## Contributing
+Icons are to be added to Bitcoin Core, via a pull request (PR), under these circumstances:
+
+- An icon used in production is being replaced with a new icon.
+- A feature is being added to the Bitcoin Core GUI that requires the addition of a new icon.
+
+If a new icon is designed for an upcoming or active PR, the designer of the icon should [open an issue](https://github.com/bitcoin-core/gui/issues/new/choose) to get feedback on the design and to make sure it is consistent with Bitcoin Core's iconography.
+
+If a proposed feature requires a new icon, the developer should [open an issue](https://github.com/bitcoin-core/gui/issues/new/choose) requesting the new icon be designed.
+
+Icons, both SVG and PNG, are not to be added to Bitcoin Core prior to a production use case.
+
+When opening a PR that intends to add an icon, a SVG source file must be included under `src/qt/res/src`, and a optimized production file (PNG) must be included under `src/qt/res/icons`. Both source and production files should be 32x32px in size.

```

Comment:
```suggestion
## Contributing
Issues and PRs pertaining to icons should be posted within the [Bitcoin Core GUI repo](https://github.com/bitcoin-core/gui). Icons should only be added to Bitcoin Core via a PR that implements the icon in its design. Icons, both SVG and PNG, are not to be added to Bitcoin Core prior to a production use case. If an icon is not being used it should be removed. 

If a developer working on a new features requires an icon not currently available in the [Bitcoin Icon set](https://github.com/BitcoinDesign/Bitcoin-Icons) they should [open an issue](https://github.com/BitcoinDesign/Bitcoin-Icons/issues) in that repo requesting an icon to be designed. 

SVG are to be added to `src/qt/res/src` directory, and a optimized production file (PNG) must be included under `src/qt/res/icons` directory.

```

Reply:
> Icons should only be added to Bitcoin Core via a PR that implements the icon in its design.

That line is redundant. The only way an icon would get in is through a PR.